### DISCUSSION TITLE
Adding a carousel

### DISCUSSION THREAD

**Discussion by Ghoral** [shadcn-ui/ui#587] [Ideas]
Hi,
I think carousel is the most used component while developing web. It would be best if there is a carouse like this. This is the only most important we are missing right now.

For example:
[https://flowbite.com/docs/components/carousel/](url)

<img width="795" alt="Screenshot 2023-06-14 at 18 51 32" src="https://github.com/shadcn/ui/assets/49188349/c0900336-ac55-446d-b84d-3f6c15eb7714">

Thank You.

**Comment by NikharPandya** [comment]
https://github.com/shadcn/ui/issues/337 discussion going on here about carousel component
